applications/ShapeOptimizationApplication/test_examples/05_mesh_independent_optimization_3D_Shell/ProjectParameters.json

```diff
@@ -0,0 +1,194 @@
+            "variable_name"   : "DISPLACEMENT",
+            "is_fixed_x"      : false,
+            "is_fixed_y"      : true,
+            "is_fixed_z"      : true,
+            "value"           : [0.0,0.0,0.0]
+        }
+    }],
+    "loads_process_list"       : [{
+        "python_module"   : "process_factory",
+        "kratos_module" : "KratosMultiphysics",
+        "check"                 : "DirectorVectorNonZero direction",
+        "help"                  : "This process ",
+        "process_name"          : "ApplyConstantVectorValueProcess",
+        "Parameters"            : {
+            "mesh_id"         : 0,
+            "model_part_name" : "PointLoad3D_load",
+            "variable_name"   : "POINT_LOAD",
+            "modulus"          : 100000.0,
+            "direction"       : [0.0,-1,0.0]
+        }
+    }],
+    "output_configuration"     : {
+        "result_file_configuration" : {
+            "gidpost_flags"       : {
+                "GiDPostMode"           : "GiD_PostBinary",
+                "WriteDeformedMeshFlag" : "WriteDeformed",
+                "WriteConditionsFlag"   : "WriteConditions",
+                "MultiFileFlag"         : "SingleFile"
+            },
+            "file_label"          : "step",
+            "output_control_type" : "step",
+            "output_frequency"    : 1.0,
+            "body_output"         : true,
+            "node_output"         : false,
+            "skin_output"         : false,
+            "plane_output"        : [],
+            "nodal_results"       : ["DISPLACEMENT","REACTION","ROTATION","POINT_LOAD"],
+            "gauss_point_results" : []
+        },
+        "point_data_configuration"  : []
+    },
+    "restart_options"          : {
+        "SaveRestart"      : false,
+        "RestartFrequency" : 0,
+        "LoadRestart"      : false,
+        "Restart_Step"     : 0
+    },
+    "constraints_data"         : {
+        "incremental_load"         : false,
+        "incremental_displacement" : false
+    },
+    "optimization_settings" : {
+        "objectives" : [{
+                "identifier"    : "strain_energy",
+                "use_kratos"    : true,
+                "gradient_mode" : "semi_analytic",
+                "step_size"     : 1e-8,
+                "discretization_weighting": true
+        }],
+        "constraints" : [],
+        "design_variables" : {
+            "design_variables_type"     : "vertex_morphing",
+            "input_model_part_name"     : "3D_Shell_diagonal",
+            "design_submodel_part_name" : "design_surface",
+            "domain_size"               : 3,
+            "filter" : {
+                "filter_function_type"       : "linear",
+                "filter_radius"              : 3,
+                "max_nodes_in_filter_radius" : 10000,
+                "matrix_free_filtering"      : false
+            },
+            "consistent_mapping_to_geometry_space": true,
+            "integration": {
```

Comment:
Would it not make more sense to put the part about the integration into the filter? Since this is only meaningful in the context of the filter?